src/checkout-resource.js

```diff
@@ -67,6 +68,30 @@ class CheckoutResource extends Resource {
       .then(handleCheckoutMutation('checkoutCreate', this.graphQLClient));
   }
 
+  /**
+   * Replaces the value of checkout's custom attributes and/or note with values defined in the input
+   *
+   * @example
+   * const checkoutId = 'Z2lkOi8vc2hvcGlmeS9DaGVja291dC9kMTZmM2EzMDM4Yjc4N=';
+   * const input = { customAttributes: [{ key: "MyKey", value: "MyValue" }] };
```

Comment:
the example should fit the rest of the examples, no spaces after or before `{` and `}`